packages/restapi/src/lib/chat/approveRequest.ts

```diff
@@ -0,0 +1,54 @@
+import axios from 'axios';
+import {
+  getAPIBaseUrls, walletToPCAIP10,
+} from '../helpers';
+import Constants from '../constants';
+import { AccountEnvOptionsType } from '../types';
+
+/**
+ *  POST '/v1/chat/request/accept
+ */
+
+export interface ApproveRequestOptionsType extends AccountEnvOptionsType {
+  senderAddress: string; // chat request sender address
+  // privateKey?: string; // private key for signature
+  status?: string;
+  sigType?: string;
+}
+
+export const approve = async (
+  options: ApproveRequestOptionsType
+) => {
+  const {
+    status = "Approved",
+    sigType = 'sigType',
+    // privateKey = null,
+    env = Constants.ENV.PROD,
+    account,
```

Comment:
senderAddress should be receiverAddress

Reply:
in approve, senderAddress is the intent sent by address

Reply:
okay okay got it